scripts/shared/post_mortem.sh

```diff
@@ -7,7 +7,7 @@ source ${SCRIPTS_DIR}/lib/utils
 
 function post_analyze() {
     echo "======================= Post mortem $cluster ======================="
-    kubectl get all -A
```

Comment:
There was no need for this change, -A is short flag for this behavior

Reply:
Yes, I know. With some versions of kubectl it's not supported. I noticed this while testing the script locally on my setup.

```
[submariner]$ kubectl get all -A
Error: unknown shorthand flag: 'A' in -A

Examples:
  # List all pods in ps output format.
  kubectl get pods
  
  # List all pods in ps output format with more information (such as node name).
  kubectl get pods -o wide
  
  # List a single replication controller with specified NAME in ps output format.
  kubectl get replicationcontroller web
  
  # List deployments in JSON output format, in the "v1" version of the "apps" API group:
  kubectl get deployments.v1.apps -o json

<SNIP>

```